internal/access/organization.go

```diff
@@ -56,14 +55,17 @@ func DeleteOrganization(c *gin.Context, id uid.ID) error {
 	return data.DeleteOrganizations(db, data.ByID(id))
 }
 
-var domainNameReplacer = regexp.MustCompile(`[^\da-zA-Z-]`)
-
 func SanitizedDomain(subDomain, serverBaseDomain string) string {
-	sanitizedDomain := domainNameReplacer.ReplaceAllStringFunc(subDomain, func(s string) string {
-		if s == " " {
-			return "-"
+	replaceSymbols := func(r rune) rune {
+		switch {
+		case r >= 'a' && r <= 'z', r >= 'A' && r <= 'Z', r >= '0' && r <= '9':
+			return r
+		case r == ' ' || r == '-':
```

Comment:
Maybe `_` as well since it might be used as a separator?

Reply:
Sounds like a good idea to me. It's different from the regex behaviour, but I'm ok to make that change if there are no concerns.

Reply:
We've updated the validation in #3066 to remove `_` from the valid character list, so I think we can omit it here.